pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2234,6 +2310,12 @@
     ::  save duct for future %give to unix
     ::
     =.  outgoing-duct.server-state.ax  duct
+    ::  send all cache bindings to runtime
+    ::
+    =/  cache-moves=(list move)
```

Comment:
I don't think this is necessary. Maybe I don't understand its purpose?

Reply:
this comes from the initial proposal, which said:

> Eyre will give a %gift to the runtime upon startup and whenever a binding is added or removed from the map.

which I guess is ambiguous about what exactly the startup %gift should contain.

Reply:
Ah, if this is happening on `born` then it totally makes sense. For some reason I thought this was happening on any HTTP request.